review/fileToReview.js

```diff
@@ -1,5 +1,4 @@
-
-var domain = "bank.local.fr"
+const domain = 'bank.local.fr';
```

Comment:
* usage of var is not a good practice since the release of ES6, in this case const is a good bet because the value won't change